### DISCUSSION TITLE
Multiple line selection and editing

### DISCUSSION THREAD

**Discussion by Dennis-Zhang-SH** [ayamir/nvimdots#589] [Q&A] [answered]
Hi, I am a newbie to vim and neovim, how to do multiple line selection and editing with same words prefix? I googled and it tells me to use `<C-v>` but didn't work.

**Comment by charliie-dev** [comment]
https://vim.fandom.com/wiki/Search_and_replace

**Comment by Dennis-Zhang-SH** [comment]
it's actually not replacing, say we have the following lines:
```
aaa;
bbb;
ccc;
```
I want to select all the semicolons and editing multiple lines, it may result as:
```
let ax = aaa_xx;
let bx = bbb_xx;
let cx = ccc_xx;
```

**Reply by charliie-dev** [reply]
Then I think macro is more suitable in this case, but TBH, I didn't learn macros that well lol
https://vim.fandom.com/wiki/Macros
Maybe others have a better solution.

**Comment by Jint-lzxy** [comment, marked as answer]
`<C-v>` then press `I` or `A` (upper case).

See also:
```vim
:help v_b_I
:help v_b_A
:help v_b_I_example
:help v_b_A_example
```

**Reply by Dennis-Zhang-SH** [reply]
> `<C-v>` then press `I` or `A` (upper case).
> 
> See also:
> 
> ```viml
> :help v_b_I
> :help v_b_A
> :help v_b_I_example
> :help v_b_A_example
> ```

press `<C-v>` would simply paste from clipboard, wouldn't enter into any mode.

**Reply by ayamir** [reply]
> press `<C-v>` would simply paste from clipboard, wouldn't enter into any mode.

It means `<C-v>` is controled by programs warpped the `nvim` like `tmux` or your terminal. `<C-v>` in vim is selection by column.

**Reply by kang8** [reply]
> press `<C-v>` would simply paste from clipboard


This is because your system key bindings are overriding the behavior of vim. Please check your settings to ensure that `<C-v>` has not been overridden, so that you can use `<C-v>` to edit multiple lines.
